aten/src/ATen/native/LinearAlgebra.h

```diff
@@ -25,5 +26,7 @@ using unpack_pivots_fn = void(*)(
 );
 DECLARE_DISPATCH(unpack_pivots_fn, unpack_pivots_stub);
 
+using linalg_solve_fn = void (*)(const at::Tensor&, const at::Tensor&, at::Tensor&, int&);
+DECLARE_DISPATCH(linalg_solve_fn, linalg_solve_sparse_csr_stub);
```

Comment:
I'm trying to add a dispatch, so that I can call the linalg_solve kernel for sparse (present in `SparseCsrTensorMath.cu` file) from `SparseCsrTensorMath.cpp` file.

Reply:
It's a good idea, with a bit more work I think you get it to do what we want: call a CUDA library for CPU inputs when CUDA is available.